draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -366,187 +369,201 @@
-      particular classifier.</t> 
-
-      <t> Metering statistics consist of meters identified by an identifier.
-      Metering counters include conform, exceed, violate, drop packets,
-      and bytes.</t>
-
-      <t>Queuing counters include instantaneous, peak, average queue length,
-      as well as output conform, exceed, tail drop packets and bytes.</t>
-
-      <t>Named statistics is defined as statistics which is tagged by a name.
-      This could be aggregated or non-aggregated.  Aggregated named
-      statistics is defined as counters which are aggregated across
-      classifier entries in a policy applied to an interface in a
-      particular direction.  Non-aggregated named statistics are counters
-      of classifier, metering or queuing which have the same tag name but
-      maintained separately.</t>
-
-      <figure>
-	<name>ietf-diffserv tree diagram</name>
-        <artwork><![CDATA[
-INSERT_TEXT_FROM_FILE(../bin/ietf-diffserv@YYYY-MM-DD-sub-tree.txt)  
-  ]]></artwork>
-      </figure>
-      <figure>
-	<name>ietf-queue-policy tree diagram</name>
-        <artwork><![CDATA[
-INSERT_TEXT_FROM_FILE(../bin/ietf-queue-policy@YYYY-MM-DD-sub-tree.txt)
-    ]]></artwork>
-      </figure>
-      <figure>
-	<name>ietf-scheduler-policy tree diagram</name>
-        <artwork><![CDATA[
-INSERT_TEXT_FROM_FILE(../bin/ietf-scheduler-policy@YYYY-MM-DD-sub-tree.txt)
-    ]]></artwork>
-      </figure>
-      <figure>
-	<name>ietf-qos-oper tree diagram</name>
-        <artwork><![CDATA[
-INSERT_TEXT_FROM_FILE(../bin/ietf-qos-oper@YYYY-MM-DD-sub-tree.txt)
-    ]]></artwork>
-      </figure>
-    </section>
+	<figure>
+	  <name>ietf-traffic-policy tree diagram</name>
+          <artwork><![CDATA[
+	  INSERT_TEXT_FROM_FILE(../bin/ietf-traffic-policy@YYYY-MM-DD-sub-tree.txt)
 
-    <section title="Modules">
-      <t>
-      Modules defined in this draft import definitions from "Common
-      YANG Data Types" <xref target="RFC6991"/> and "A YANG Data Model
-      for Interface Management" <xref target="RFC8343"/>.
-      </t>
+	  ]]></artwork>
+	</figure>
 
-      <section title="ietf-traffic-policy">
         <figure>
 	  <name>ietf-traffic-policy module</name>
           <artwork><![CDATA[
-<CODE BEGINS> file "ietf-traffic-policy@YYYY-MM-DD.yang"
-INSERT_TEXT_FROM_FILE(../bin/ietf-traffic-policy@YYYY-MM-DD.yang)
-<CODE ENDS>
-    ]]></artwork>
+	  <CODE BEGINS> file "ietf-traffic-policy@YYYY-MM-DD.yang"
+	  INSERT_TEXT_FROM_FILE(../bin/ietf-traffic-policy@YYYY-MM-DD.yang)
+	  <CODE ENDS>
+	  ]]></artwork>
         </figure>
       </section>
 
-      <section title="ietf-qos-action">
+      <section title="QoS Action Module">
+	<t>QoS Action module includes actions such as marking,
+	metering (Note, all I see is marking functions, no metering
+	functions in the module), or queuing. Metering and marking
+	actions are realized by augmenting the QoS Policy (Note, there
+	is nothing called a QoS Policy module any more) module. Any
```

Comment:
yes, we can say Traffic-Policy .. AQM is Active Queue Management like RED.. should add abbreviations ..